projects/tinkerbell/hegel/Makefile

```diff
@@ -0,0 +1,27 @@
+BASE_DIRECTORY=$(shell git rev-parse --show-toplevel)
+GIT_TAG?=$(shell cat GIT_TAG)
+GOLANG_VERSION?="1.15"
+
+REPO=hegel
+REPO_OWNER=tinkerbell
+
+BASE_IMAGE_NAME?=eks-distro-minimal-base
+
+BINARY_TARGET_FILES=hegel
+SOURCE_PATTERNS=./cmd/hegel
+
+FIX_LICENSES_STRPTIME_TARGET=$(REPO)/vendor/github.com/pbnjay/strptime/LICENSE.txt
+
+IMAGE_USERADD_USER_NAME=tinkerbell
```

Comment:
Not sure if it's a cause for concern (probably not) but Tinkerbel runs `useradd` with the `-D` option to use defaults without creating a new user/group.
```Dockerfile
RUN adduser -D -u 1000 tinkerbell
```

Reply:
Yea I had this doubt as well. But i saw in the build logs that its getting set to default?

`
#10 [builder 5/5] RUN groupadd -r tinkerbell &&     useradd -r -g tinkerbell -u 1000 tinkerbell
#10 14.96 useradd: group '100' does not exist
#10 14.96 useradd: the GROUP= configuration in /etc/default/useradd will be ignored
#10 DONE 23.3s
`